src/utils/episodeCode.test.ts

```diff
@@ -0,0 +1,50 @@
+import { episodeCode } from "./episodeCode";
+
+test("episode card adds zero padding to one digit numbers", () => {
+  expect(
+    episodeCode({
+      id: 4952,
```

Comment:
Might be more readable to have this object stored as a variable outside of the assertion